lib/async_hooks.js

```diff
@@ -71,8 +71,7 @@ function fatalError(e) {
     Error.captureStackTrace(o, fatalError);
     process._rawDebug(o.stack);
   }
-  if (process.execArgv.some(
-      (e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
+  if (process.execArgv.some((e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
```

Comment:
Thanks for the info!

I think it would still be nice to move the expression up in it's own variable though, since it prevents the RegExp from being recreated multiple times.